### PR TITLE
satellite/admin: Add Client App and more functions

### PR DESCRIPTION
**Pull request by stefanbenten**
What: adds a client side tool called storj-admin to manage requests via an UI and adds more crucial functions to the admin API.

Why: It helps us managing the satellite
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: cmd/storj-admin/template/template.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+package template
+
+import (
+	"html/template"
+)
+
+var (
+	Public = template.Must(
+		template.New("public").Parse(string(MustAsset("public.html.tmpl"))),
```
Comment (egonelbre) [inline cmd/storj-admin/template/template.go:14]: There's only one file, you can write html in Go code directly.

File: satellite/admin/README.md [hunk 2]
```diff
@@ -4,7 +4,29 @@ Satellite Admin package provides API endpoints for administrative tasks.
 
 Requires setting `Authorization` header for requests.
 
-## GET /api/user/{user-email}
+## POST /api/user
```
Comment (egonelbre) [inline satellite/admin/README.md:7]: Let's make separate PR-s for each addition, that way it's clearer to see the tests etc.

File: satellite/admin/user.go [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+		http.Error(w, "Unable to create UUID", http.StatusInternalServerError)
+		return
+	}
+
+	user, err := server.db.Console().Users().Insert(ctx, &console.User{
+		ID:           userID,
+		Email:        input.Email,
+		FullName:     input.Name,
+		PasswordHash: hash,
+		Status:       1,
```
Comment (egonelbre) [inline satellite/admin/user.go:74]: Do we have constants for these?

File: satellite/admin/README.md (hunk 2 above)
Comment (mniewrzal) [reply satellite/admin/README.md:7]: yeah, its 1200+ lines PR right now

File: cmd/storj-admin/template/template.go (hunk 1 above)
Comment (stefanbenten) [reply cmd/storj-admin/template/template.go:14]: I am using this route as i plan on splitting the pages into subtemplates to make them better scoped.

File: satellite/admin/README.md (hunk 2 above)
Comment (stefanbenten) [reply satellite/admin/README.md:7]: I'll do that.

**Comment by stefanbenten** [comment]
Closing in favor of the swagger client side tools.
